EmailFunctionResearch.md

```diff
@@ -44,3 +44,7 @@ print("Message sent!")
 send_email(subject, body, sender, recipients, password)
 
 Once this is all implemented, users should be able to freely email themselves summaries!
+
+Email: textinsights@gmail.com
+PW: SoftInnoStudio2023%
+App Password: lgoi fdyt jvin auaj
```

Comment:
Brah put this in an env file?? rip it's already been committed but this should be in an env file 🤡